### DISCUSSION TITLE
The shortcut for deselect all mod

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#18186] [Q&A] [answered]
`1` key is now not able to deselect all mod, is this expected?
It used to be available in previous versions and there's no reason to remove it

**Comment by bdach** [comment]
It's an oversight, but I'm not sure how to feel about "1" being the binding for this... If I were to bring back, then I'd probably use actual platform shortcuts (ctrl+A, ctrl+D). Is this fine?

**Reply by cdwcgt** [reply]
Ctrl+D is already bind for beatmap list
![image](https://user-images.githubusercontent.com/34775378/167427201-45a240fe-eaab-4c9e-9872-66b8e3b63ace.png)
and be honest, many players are used to using 1 to deselect. And 1 is no other use under this interface(unbinded)
Although counter-intuitive to the interface itself, consider putting `deselect all` button on the left?

**Reply by frenzibyte** [reply]
The keybind for toggling overlays will change according to https://github.com/ppy/osu/issues/12776#issuecomment-840287245.

**Reply by peppy** [reply]
> It's an oversight, but I'm not sure how to feel about "1" being the binding for this... If I were to bring back, then I'd probably use actual platform shortcuts (ctrl+A, ctrl+D). Is this fine?

I was about to be like "oh wow, i didn't know ctrl+D existed!" and tried on windows to confirm. It deletes files. I don't think there is any precedent for ctrl+D deselecting.

My proposal would be a new key binding with a sane key like `Delete` maybe? Then in the first run overlay we can add key binding mutation for "stable" users.

**Reply by bdach** [reply]
> I don't think there is any precedent for ctrl+D deselecting.

I think Photoshop does this and image editors copied that. I'll admit I didn't check what the windows shell does.

**Reply by frenzibyte** [reply]
On GIMP, it's `Ctrl+Shift+A` that deselects, and it feels natural adding shift to perform the opposite of what a keybind does, but perhaps not the case for many people.

**Reply by Moltanicaa** [reply]
From the side of the user experience (subjective opinion), it looks strange to replace one press of "1" with a whole combination of Ctrl + something.

**Reply by bdach** [reply]
Well I think the "1" binding was strange to begin with. But if it's going to be rebindable then /shrug.

**Comment by frenzibyte** [comment, marked as answer]
Tracking in https://github.com/ppy/osu/issues/18235
